src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -132,6 +132,6 @@
     <DefaultRunAsPolicy UserRef="MaestroUser" />
   </Policies>
   <Certificates>
-    <EndpointCertificate X509FindType="FindBySubjectName" X509FindValue="[WebCertName]" Name="WebSslCert" />
```

Comment:
This is one of those changes that VS makes, right? I'm not sure if it should be included here.

Reply:
Stupid vs. I forgot to und that. Thanks :)